models/quality_measures/intermediate/nqf2372_breast_cancer_screening/quality_measures__int_nqf2372_numerator.sql

```diff
@@ -130,43 +131,44 @@ with denominator as (
           denominator.patient_id
         , denominator.performance_period_begin
         , denominator.performance_period_end
+        , denominator.performance_period_lookback
         , denominator.measure_id
         , denominator.measure_name
         , denominator.measure_version
         , case
             when qualifying_claims.claim_start_date
-                between denominator.performance_period_begin
+                between denominator.performance_period_lookback
```

Comment:
Replacing "period begin" with "period lookback" since the period begin date is now a calendar year. This lookback is for the 27 months allowed for mammograms.